.evergreen/build_snapshot_rpm.sh

```diff
@@ -89,14 +89,12 @@ sudo mock -r ${config} --use-bootstrap-image --isolation=simple --init
 mock_root=$(sudo mock -r ${config} --use-bootstrap-image --isolation=simple --print-root-path)
 sudo mock -r ${config} --use-bootstrap-image --isolation=simple --install rpmdevtools git rpm-build cmake python python3-sphinx gcc openssl-devel
 sudo mock -r ${config} --use-bootstrap-image --isolation=simple --copyin "$(pwd)" "$(pwd)/${spec_file}" /tmp
-if [ ! -f VERSION_CURRENT ]; then
```

Comment:
Nice, it's better to use calc_release_version.py rather than rely on the hard coded version anyway. Curious, was this necessary to work around an issue? I think it is hard coded to `1.18.0` based on https://github.com/mongodb/mongo-c-driver/blob/master/.evergreen/config.yml#L29:L29, and `1.17.0-pre` on the `r1.17` branch.

Reply:
Right.  The issue that hard-coding of VERSION_CURRENT is meant to work around is the age of Git (and possibly Python) on some of the older platforms, making the determination of the version from the Git history problematic. However, when we build the RPM snapshot packages, we always do so in a very recent chroot environment, so we can actually just calculate the correct version.  Additionally, we can't just use {{x.y.z-pre}} in the spec file, because the "-" is not a valid character in that part of the version string.  Hence, it is better to calculate it in the way the snapshot script does it for building the RPMs.